src/goCheck.ts

```diff
@@ -55,22 +55,34 @@ function runTool(args: string[], cwd: string, severity: string, useStdErr: boole
 				outputChannel.appendLine(['Finished running tool:', cmd, ...args].join(' '));
 
 				let ret: ICheckResult[] = [];
+				let unexpectedOutput = false;
+				let atleastSingleMatch = false;
 				for (let i = 0; i < lines.length; i++) {
 					if (lines[i][0] === '\t' && ret.length > 0) {
 						ret[ret.length - 1].msg += '\n' + lines[i];
 						continue;
 					}
 					let match = /^([^:]*: )?((.:)?[^:]*):(\d+)(:(\d+)?)?:(?:\w+:)? (.*)$/.exec(lines[i]);
 					if (!match) {
-						if (printUnexpectedOutput) outputChannel.appendLine(lines[i]);
+						if (printUnexpectedOutput) unexpectedOutput = true;
 						continue;
 					}
+					atleastSingleMatch = true;
 					let [_, __, file, ___, lineStr, ____, charStr, msg] = match;
 					let line = +lineStr;
 					file = path.resolve(cwd, file);
 					ret.push({ file, line, msg, severity });
 					outputChannel.appendLine(`${file}:${line}: ${msg}`);
 				}
+				if (!atleastSingleMatch && unexpectedOutput && useStdErr && vscode.window.activeTextEditor) {
+					outputChannel.appendLine(stderr);
+					ret.push({
+						file: vscode.window.activeTextEditor.document.fileName,
+						line: 1,
+						msg: 'Something went wrong. Check the Go channel in the output window',
```

Comment:
Oh man I did that thing again where I typed that this morning before standup but didn't submit the review. Just had to say that since we talked about this at standup.